src/launch/launcher.rs

```diff
@@ -26,4 +29,20 @@ impl<'a, U: AsRawFd, V: AsRawFd> Launcher<'a, New, U, V> {
 
         Ok(launcher)
     }
+
+    pub fn start(self, start: Start) -> Result<Launcher<'a, Started, U, V>> {
+        let mut launch_start = LaunchStart::new(&start.policy, &start.cert, &start.session);
+        LAUNCH_START.ioctl(
+            self.kvm,
+            &mut Command::from_mut(self.sev, &mut launch_start),
+        )?;
```

Comment:
```suggestion
        let mut cmd = Command::from_mut(self.sev, &mut launch_start);
        LAUNCH_START.ioctl(self.kvm, &mut cmd)?;
```

I think this is more readable.